modules/preparing-aws-sts-credentials-for-oadp.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-aws-sts/oadp-aws-sts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="preparing-aws-sts-credentials-for-oadp_{context}"]
+= Preparing AWS STS credentials for OADP
+
+An {aws-full} account must be prepared and configured to accept an {oadp-first} installation. Prepare the {aws-short} credentials by following the proceeding steps.
+
+.Procedure
+. Define the `cluster_name` environment variable by running the following command:
++
+[source,terminal]
+----
+$ export CLUSTER_NAME= <AWS cluster name> <1>
+----
+<1> The variable can be set to any value.
```

Comment:
Should there be a + after this?

Reply:
No. The <1> is a call-out.

Reply:
I don't think we usually do that.

Reply:
Not a +. My bad. I mean an empty line.

So...

Line 18: call out
Line 19: empty line
Line 20: . Retrieve all the details...

It may not matter, or it may not render properly in the Customer Portal without an empty line between the call out and the next step.

Reply:
@gwynnemonahan 
Thank you for the clarification.
I think we both got confused, because I was looking at a file with a similar name.
I added the space as you suggested.
Thanks!